dotcom-rendering/src/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -319,8 +319,8 @@ const RemoteBanner = ({
 		debounce: true,
 	});
 
-	useOnce(() => {
-		if (module === undefined || meta === undefined) {
+	useEffect(() => {
+		if (meta === undefined) {
```

Comment:
Why did we remove the `module === undefined` check?

Reply:
TypeScript told me it could not be undefined :)

Reply:
This PR will also make the `meta` check unnecessary:
https://github.com/guardian/dotcom-rendering/pull/9217